test/vectorize_test.exs

```diff
@@ -0,0 +1,56 @@
+defmodule AshAiVectorizeTest do
+  use AshAi.RepoCase, async: false
+
+  use Oban.Testing, repo: AshAi.TestRepo, prefix: "private"
+
+  alias AshAi.Test.Music
+
+  test "after_action strategy works as expected" do
+    artist =
+      Music.create_artist_after_action!(%{
+        name: "John Doe",
+        bio: "John Doe is a musician"
+      })
+      |> Ash.load!([:full_text_vector])
+
+    assert is_list(Ash.Vector.to_list(artist.full_text_vector))
+  end
+
+  test "manual strategy works as expected" do
+    artist =
+      Music.create_artist_manual!(%{
+        name: "John Doe",
+        bio: "John Doe is a musician"
+      })
+      |> Ash.load!([:full_text_vector])
+
+    assert is_nil(artist.full_text_vector)
+
+    artist = Music.update_embeddings_artist_manual!(artist) |> Ash.load!([:full_text_vector])
+    assert is_list(Ash.Vector.to_list(artist.full_text_vector))
+  end
+
+  test "ash_oban strategy works as expected" do
+    Oban.start_link(AshOban.config([AshAi.Test.Music], Application.get_env(:ash_ai, :oban)))
+
+    artist =
+      Music.create_artist_oban!(%{
```

Comment:
Can we add a test that updating works? You can use `AshOban.Test.schedule_and_run_triggers` to run the necessary triggers & jobs.

Reply:
Or actually just `Oban.drain_queues` like you already are since the trigger isn't using cron.